BaseFilter/sections/specific.txt

```diff
@@ -343,6 +343,8 @@ $xmlhttprequest,third-party,denyallow=rsc.cdn77.org|linkvertise.com|fastly.net|s
 !
 ! NOTE: Specific rules
 !
+celebfapper.com##.row
```

Comment:
Dangerous rule in generic. Also only seeing one element with some elements in code.

Reply:
Remove this rule, it is not needed. Base filter already block it